views/default/navigation/tabs.php

```diff
@@ -19,6 +19,31 @@
 		continue;
 	}
 	
+	// BC tab params
+	if (!isset($tab['text']) && isset($tab['title'])) {
+		$tab['text'] = $tab['title'];
```

Comment:
Maybe put deprecation notices on all of these?

Reply:
done